website/docs/docs/build/semantic-models.md

```diff
@@ -105,6 +105,29 @@ semantic_models:
         type: categorical
 ```
 
+<VersionBlock firstVersion="1.7">
+
+Semantic models support configs in either the schema file or at the project level. 
```

Comment:
non-blocker question - are we still using schema file or properties.yml file per [this doc](https://docs.getdbt.com/reference/configs-and-properties#where-can-i-define-properties)? I'm a little confused as to what to use